stratum/hal/lib/barefoot/bf_chassis_manager.cc

```diff
@@ -351,6 +391,14 @@ ::util::Status BFChassisManager::PushChassisConfig(
 
 ::util::Status BFChassisManager::VerifyChassisConfig(
     const ChassisConfig& config) {
+  std::map<uint64, int> node_id_to_unit;
```

Comment:
This map is not used anywhere.

Reply:
It is used below in line 429

Reply:
Isn't that `node_id_to_unit_`, the class member with the `_` at the end?

Reply:
This is a good catch. We are in Verify, so we shouldn't be using the class member in this function. Updating the call in 429 to use the local variable.